nixos/modules/services/mail/postfix.nix

```diff
@@ -466,16 +466,17 @@ in
         ";
       };
 
-      sslCert = mkOption {
+      tlsTrustedAuthorities = mkOption {
```

Comment:
Since this option is only use to set `smtp_tls_CAfile`, maybe it would be easier to just call it `smtpTlsCAfile` in order to have a 1 to 1 mapping with the postfix configuration. We could even mention this postfix variable documentation in the description.

Reply:
>Since this option is only use to set smtp_tls_CAfile

It's also used to enable enforce tls for outgoing connections in case no ssl certificate was set.

> maybe it would be easier to just call it smtpTlsCAfile in order to have a 1 to 1 mapping with the postfix configuration.

The problem I see that it would introduces a new naming schema. There's no other option with a smtp{,d} prefix.

> We could even mention this postfix variable documentation in the description.

I'll updating the PR tonight referencing the postfix config value.

Reply:
Updated the description.

Reply:
Ok. Let's use `tlsTrustedAuthorities`!